src/main/sarl/fr/utbm/info/ia51/crowdproject/motion/MotionAlgorithms.sarl

```diff
@@ -43,9 +47,23 @@ class KinematicWanderAlgorithm implements WanderAlgorithm {
  */
 class SteeringWanderAlgorithm implements WanderAlgorithm {
 
-	override run(emitter : UUID, currentPosition : Vector2) : MotionQuery {
-		val query = new MotionQuery(emitter, MotionBehavior::STEERING)
-		// TODO
-		return query
+	override run(emitter : UUID, currentPosition : Vector2, orientation : double, velocity : Vector2) : MotionQuery {
```

Comment:
uh.... I need more information to pass if i want the velocity : the speed is not supposed to be constant for example so i need it to calculate the velocity. If I remove the velocity, I'll need other things to pass...